docs/status.md

```diff
@@ -84,7 +84,7 @@ one of the following tracking labels.
 | dynamic_iota             | no            | revisit      | infeasible     | yes             | no          |
 | dynamic_pad              | no            | revisit      | no             | yes             | no          |
 | dynamic_reshape          | no            | revisit      | infeasible     | yes             | no          |
-| dynamic_slice            | no            | yes*         | yes*           | yes             | no          |
+| dynamic_slice            | yes           | revisit      | yes            | yes             | no          |
```

Comment:
Based on the decision in 
https://github.com/openxla/stablehlo/issues/557

maybe we can have a "yes" in verification?

Reply:
Superceded by [commnent](https://github.com/openxla/stablehlo/pull/556#discussion_r1024719886) below.

Reply:
Based on the updates to the PR, it looks like #557 can be closed and Verification can be switched to "yes"?

Reply:
Sorry, I've recycled the same issue to track Sandeep's comment. As you mentioned in #557, the `start_indices` should check that all indices are of the same type since we are going the route of allowing integers, so I left this as `revisit`.

Reply:
Okay got it - sorry got confused about the outcome of the discussion. Looks like we decided that `start_indices` should be tensors of integer type (https://github.com/openxla/stablehlo/pull/556#discussion_r1024719886). 

If I understand this correctly, then we need to do the following follow-ups:
  1. Change Inputs to "variadic number of 0-dimensional tensors of integer type".
  2. Change Constraints to add " * (C3) All `start_indices` have the same type".
  3. Update the first comment in #557 to reflect the follow-up needed.

Reply:
Thanks, it is fixed in the recent commit. As for 3, since it pertains to DynamicUpdateSlice, I'll also update that PR since I don't have that constraint listed (and keep #557 open and status `revisit`).